lightfm-port/php/Evaluation.php

```
@@ -0,0 +1,257 @@
+     * 
+     * @param LightFM $model Trained LightFM model
+     * @param CSRMatrix $testInteractions Test interactions matrix
+     * @param int $k Number of top recommendations to consider
+     * @param CSRMatrix|null $trainInteractions Training interactions (to exclude from evaluation)
+     * @return float Mean precision@k across all users
+     */
+    public static function precisionAtK(
+        LightFM $model, 
+        CSRMatrix $testInteractions, 
+        int $k = 10,
+        ?CSRMatrix $trainInteractions = null
```

PHPMD (2.15.0)

Avoid unused parameters such as '$trainInteractions'. (Unused Code Rules)

(UnusedFormalParameter)